packages/backend/src/templates/quarkus-langchain4j.mustache

```diff
@@ -0,0 +1,36 @@
+application.properties
+======================
+quarkus.langchain4j.openai.base-url={{{ baseUrl }}}
+quarkus.langchain4j.openai.api-key=sk-dummy
+
+pom.xml
+=======
+<dependency>
+  <groupId>io.quarkiverse.langchain4j</groupId>
+  <artifactId>quarkus-langchain4j-core</artifactId>
+  <version>0.9.1</version>
```

Comment:
is there a way to grab the latest version automatically without updating the template ?

Reply:
See last commit